walk-in-interview/src/main/webapp/index.html

```diff
@@ -14,6 +14,7 @@
     <button id="new-post"></button>
     <div id="page-title"></div>
     <button id="account"></button>
+    <button id="log-in"></button>
```

Comment:
isn't the 'account' button the same as the 'log-in' button?

Reply:
"account" button is to create a new account, while "log-in" button is to sign in existing account

Reply:
maybe you want to rename "account" to "sign-up" and "log-in" to "sign-in" ?